apps/core/models/snippets.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+from django.utils.translation import gettext as _
+from modelcluster.fields import ParentalKey
+from modelcluster.models import ClusterableModel
+from wagtail.admin.panels import FieldPanel, InlinePanel
+from wagtail.fields import RichTextField
+from wagtail.models import Orderable
+from wagtail.snippets.models import register_snippet
+
+
+@register_snippet
+class FooterContent(ClusterableModel):
```

Comment:
I think it'd be nice to have a `__str__` method here that just returns `"Footer content"`, so that it doesn't say `FooterContent object (1)` in the admin